core/services/ocr2/delegate.go

```diff
@@ -749,6 +753,8 @@ func (d *Delegate) newServicesMercury(
 		return nil, ErrRelayNotEnabled{Err: err, Relay: spec.Relay, PluginName: "mercury"}
 	}
 
+	// TODO: can we somehow insert d.bridge into the provider here?
+	// Or into the transmitter once the provider is created?
```

Comment:
@cedric-cordenier suggestions?

Reply:
@bolekk We should add the bridge via the relayer constructor; I'm working on the signatures PR which will add this